### DISCUSSION TITLE
How to OVERRIDING SYSTEM VALUE in supabase?

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#2837] [Questions] [answered]
**Error
{hint: "Use OVERRIDING SYSTEM VALUE to override.",…}
code: "428C9"
details: "Column \"id\" is an identity column defined as GENERATED ALWAYS."
hint: "Use OVERRIDING SYSTEM VALUE to override."
message: "cannot insert into column \"id\""

**Comment by soedirgo** [comment, marked as answer]
You can't explicitly set a value for a `GENERATED ALWAYS` column. Try running:
```sql
ALTER TABLE <your_table> ALTER <your_column> SET GENERATED BY DEFAULT;
```
